update.go

```diff
@@ -55,8 +57,23 @@ func update(upstream string, branch string, downstream string,
 		Name: upstreamDefaultRemoteName,
 		URLs: []string{upstream},
 	})
-	// TODO: error check for remote
+	if err != nil {
+		switch err {
+		case git.ErrRemoteNotFound:
+			log.Fatal("remote not found")
+		default:
+			log.Println(err)
+		}
+	}
 
+	pull(r, upstream, upstreamRef, upstreamAuth)
+
+	push(r, downstream, upstreamRef, downstreamRef, downstreamAuth)
```

Comment:
Might be good to just give a []RefSpec to pull and push. That would simplify things later on.